build.gradle

```diff
@@ -111,27 +107,24 @@ dependencies {
     testImplementation "org.apache.flink:flink-streaming-java_${scalaBinaryVersion}:${flinkVersion}:tests"
 
     testRuntimeOnly "com.vladsch.flexmark:flexmark-all:0.35.10"
-    testRuntime 'org.pegdown:pegdown:1.4.2'
-    testCompile "info.picocli:picocli:4.2.0"
+    testRuntimeOnly 'org.pegdown:pegdown:1.4.2'
+    testImplementation "info.picocli:picocli:4.2.0"
```

Comment:
Why did you need to change these?

Reply:
The previous variables were deprecated and these are the new names of them.